realsense2_camera/CMakeLists.txt

```diff
@@ -130,12 +130,16 @@ elseif("$ENV{ROS_DISTRO}" STREQUAL "eloquent")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DELOQUENT")
   set(SOURCES "${SOURCES}" src/ros_param_backend_foxy.cpp)
 elseif("$ENV{ROS_DISTRO}" STREQUAL "foxy")
-  message(STATUS "Build for ROS2 foxy")
+  message(STATUS "Build for ROS2 Foxy")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DFOXY")
   set(SOURCES "${SOURCES}" src/ros_param_backend_foxy.cpp)
 elseif("$ENV{ROS_DISTRO}" STREQUAL "galactic")
-  message(STATUS "Build for ROS2 galactic")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DFOXY")
```

Comment:
@charlielito not sure about why this flag was set to `DFOXY` Here. Was this on purpose?

Reply:
No, it is a mistake :sweat_smile: